src/Components/Facility/models.tsx

```diff
@@ -566,6 +566,7 @@ export interface BaseUserModel {
   last_name: string;
   user_type: string;
   last_login: string;
+  read_profile_image_url: string;
```

Comment:
are you sure the variable name is correct? 

```suggestion
  read_profile_picture_url: string;
```